app/cfg/config.sample.json

```diff
@@ -1,6 +1,6 @@
 {
-    "SQLALCHEMY_DATABASE_URI": "",
+    "SQLALCHEMY_DATABASE_URI": "mariadb+mariadbconnector://user:password@localhost:5000/mysql",
```

Comment:
Nitpick, can you change `mysql` to `table_name` or something, to better signify that that's the table name, and not some other mysql identifier or something.

Reply:
nice suggestion!

Reply:
```suggestion
    "SQLALCHEMY_DATABASE_URI": "mariadb+mariadbconnector://user:password@localhost:5000/table_name",
```